### PR TITLE
chore: setup repo configuration

### PR DESCRIPTION
**Pull request by davidtorres**

### CODE REVIEW

File: .kokoro/build.sh [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  bash ${KOKORO_GFILE_DIR}/codecov.sh
+fi
+
+# fix output location of logs
+bash .kokoro/coerce_logs.sh
+
+if [[ "${ENABLE_FLAKYBOT}" == "true" ]]
+then
+    chmod +x ${KOKORO_GFILE_DIR}/linux_amd64/flakybot
+    ${KOKORO_GFILE_DIR}/linux_amd64/flakybot -repo=googleapis/java-pubsublite
```
Comment (acocuzzo) [inline .kokoro/build.sh:131]: Please replace java-pubsublite with managedkafka here and elsewhere

**Review by acocuzzo — CHANGES_REQUESTED** [review]
Please replace `java-pubsublite` with `managedkafka` everywhere, thanks! Otherwise looks good.

File: .github/trusted-contribution.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  - "renovate[bot]"
+  - "forking-renovate[bot]"
+  - "release-please[bot]"
+annotations:
+  # Trigger Cloud Build tests
+  - type: comment
+    text: "/gcbrun"
+  # Trigger Kokoro example
+  # - type: label
+  #   text: "kokoro:force-run"
```
[the remark this answers is not in the document]
Comment (davidtorres) [reply .github/trusted-contribution.yml:32]: I just copied these files from the devrel template https://github.com/GoogleCloudPlatform/sample-repo-template/blob/main/.github/trusted-contribution.yml

**Comment by davidtorres** [comment]
> Please replace `java-pubsublite` with `managedkafka` everywhere, thanks! Otherwise looks good.

Actually just reverted the .kokoro folder, it is not needed for this repo yet.

**Review by acocuzzo — APPROVED** [review]

File: .github/trusted-contribution.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (acocuzzo) [reply .github/trusted-contribution.yml:32]: Gotcha, sounds good!

**Pull request merged**